src/main/index.js

```diff
@@ -86,6 +86,21 @@ app.on("ready", async () => {
     loadWindow();
   });
 
+  ipcMain.handle("setLibcorePassword", async (event, { password }) => {
+    try {
+      await internal.setPassword(password);
+      if (!internal.process) {
+        await internal.start();
+      }
+    } catch (e) {
+      throw new Error("wrong password");
```

Comment:
should we make a custom error for this? it's displayed or used programmatically no?